integration-commons/pom.xml

```diff
@@ -55,5 +55,9 @@
             <groupId>commons-codec</groupId>
             <artifactId>commons-codec</artifactId>
         </dependency>
+        <dependency>
```

Comment:
The problem is that I have moved the RsaKeyUtils to commons/utils, since it is an utility class. And there is a  method that return an OAuthRsaSigner, that is a google-oauth-client class. What should I do in this case?